cmd/api/src/api/middleware/auth_test.go

```diff
@@ -83,7 +87,121 @@ func TestPermissionsCheck(t *testing.T) {
 				Session: model.UserSession{},
 			},
 		}).
-		OnHandler(permissionsCheckHandler(handlerReturn200, auth.Permissions().AuthManageSelf)).
+		OnHandler(permissionsCheckAllHandler(handlerReturn200, auth.Permissions().AuthManageSelf)).
+		Require().
+		ResponseStatusCode(http.StatusForbidden)
+
+	test.Request(t).
+		WithURL("http//example.com").
+		WithHeader(headers.RequestID.String(), "requestID").
+		WithContext(&ctx.Context{
+			AuthCtx: auth.Context{
+				PermissionOverrides: auth.PermissionOverrides{},
+				Owner: model.User{
+					Roles: model.Roles{
+						{
+							Name:        "Big Boy",
+							Description: "The big boy.",
+							Permissions: auth.Permissions().All(),
+						},
+					},
+				},
+				Session: model.UserSession{},
+			},
+		}).
+		OnHandler(permissionsCheckAllHandler(handlerReturn200, auth.Permissions().AuthManageSelf)).
+		Require().
+		ResponseStatusCode(http.StatusOK)
+}
+
+func TestPermissionsCheckAtLeastOne(t *testing.T) {
```

Comment:
For my understanding when is it conventional for the tests to include `_` as some follow that but these do not? 🤔

Reply:
Tests that target a receiver function such as `func (Type *s) Receiver` would then result in the test name `TestType_Receiver`.

Reply:
I'm going to leave these as is unless someone strongly disagrees and also tells me what I might rename these tests to.